File: Assets/Scripts/UnityScriptTest.cs
using UnityEngine;
using System.Collections;

public class UnityScriptTest : MonoBehaviour {

	private int _run = 4;

	// Use this for initialization
	private void Start () {
	
	}
	
	// Update is called once per frame
	private void Update () {
	
	}

	public int run
	{
		set { _run = value; }
		get { return _run; }
	}

	private void OnGUI()
	{
		if(GUI.Button(new Rect(200,10,100,30), "C#"))
		{
			run = 20;
			Debug.Log(run);
		}
	}
}

File: Assets/Scripts/UnityScriptTest.js
#pragma strict

class UnityScriptTest extends MonoBehaviour 
{
	private var _run:int = 4;
	public var gg:int = 10;
	public static var momo:int = 10;
	public function get run():int
	{
		return _run;
	}

	public function set run(value)
	{
		_run = value;
	}
	
	function Start () {

	}
	function Update () {

	}
	
	function OnGUI()
	{
		if(GUI.Button(Rect(10,10,100,30), "uScript"))
		{
			run = 30;
			Debug.Log(run);
		}
	}
}